README.md

```diff
@@ -1,3 +1,38 @@
-# Moeller Lab metagenomics processing pipeline
+# Moeller Lab Metagenomics Processing Pipeline
 
+## Overview
 Snakemake pipeline for basic processing of metagenomic data from the lab.
+
+Module 1: **sn-mg-QC**
+
+Inputs:
+  - Directory of raw fastq files from samples
+  - `File_Paths.txt` document, with 3 columns: `Sample_ID`, `R1`, `R2`
+    - Files will be renamed with `Sample_ID`
+    - `R1` and `R2` should be the full path to the forward and reverse reads, respectively
+  - Host genome in fasta format or bowtie2 index thereof
```

Comment:
Another stretch goal: could optionally give it an NCBI assembly ID and have it automatically download the appropriate reference. NCBI should have pre-computed indices for most available genomes.

Reply:
I do like this idea. It probably isn't too difficult to implement either. I'll update the README to reflect this, but I guess it is just something that can be specified/described in the `config.yaml` file.